app/services/association_authorizer.rb

```diff
@@ -1,20 +1,21 @@
 class AssociationAuthorizer
-  def self.authorized_associations(associations,  klass_name = nil, should_raise_exception = true)
+  def self.authorized_associations(associations,  klass_name = nil, should_raise_exception = true, action = 'view')
```

Comment:
Np, sorry for confusing, just hoping if someone else needs something similar in the future they use this and not break a plugin by dynamically creating the permission name.